lib/plugins/aws/customResources/resources/cognitoUserPool/lib/permissions.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const AWS = require('aws-sdk');
```

Comment:
We may narrow require to:

```javascript
const Lambda = require('aws-sdk/clients/lambda');
```

While it won't make much difference now. It's a way to reduce size of a lambda, if packaging is setup to take only modules that are required (and I hope in near future we'll have that way integrated into core)